shared/tsconfig.base.json

```diff
@@ -1,9 +1,8 @@
 {
   "compilerOptions": {
-    "outDir": "${configDir}/build",
-    "noEmit": true,
+    "outDir": "${configDir}/node_modules/.cache/.tsc_cache/build",
     "strict": true,
-    "composite": false,
+    "composite": true,
```

Comment:
Now set `composite` to `true` for all projects. This is required for project references to work.